app/view_models/prior_authority/v1/application_details.rb

```diff
@@ -74,18 +71,16 @@ def corrections_requested?
         updates_needed.include?(SendBackForm::INCORRECT_INFORMATION)
       end
 
-      delegate :latest_provider_update_event, to: :submission
-
       def provider_added_further_information?
-        latest_provider_update_event.details['comment'].present?
+        submission.data['further_information'].any? { _1['new'] }
```

Comment:
Todo: Migrations to ensure that `new` attributes are present.

Reply:
Is that TODO to be part of this PR?